tools/cd_scripts/e2e_test.sh

```diff
@@ -169,18 +177,16 @@ sudo -u starterscriptuser bash -c '
 set -e
 # Print commands and their arguments as they are executed.
 set -x
-
+export KOKORO_ARTIFACTS_DIR=/home/starterscriptuser/failure_logs
```

Comment:
Can we write comment on why we are setting this env variable as it seems confusing to use kokoro in this script